.github/pull_request_template.md

```diff
@@ -1,25 +1,28 @@
 TODO: Please describe your changes here.
 
-TODO: Delete the following lines but _keep and fill out_ (!) the checklist below.
+<!--
+TODO: Please read the following on ICU Contributing
 
-Thank you for your pull request!\
-General info on contributing: Please see https://github.com/unicode-org/icu/blob/main/CONTRIBUTING.md
+Thank you for your pull request! 
 
-Contributors license agreement (CLA):
-- You will be automatically asked to sign the CLA before the PR is accepted.
-- To sign the CLA: https://cla-assistant.io/unicode-org/icu
+* For general info on contributing: https://github.com/unicode-org/icu/blob/main/CONTRIBUTING.md
+* Associating PRs to Jira issues
```

Comment:
is it "to" or "with?

Reply:
Perhaps I was thinking in "IT vernacular" in which maps associate keys to values, and PRs must map to a Jira ticket number: https://english.stackexchange.com/a/78404

I'm fine changing it if you prefer.

Reply:
Associating ... to ... sounds weird to me, but you are the native speaker :-)
stackexchange says “... in general purpose usage, "Associated with" is preferable nearly every time.”